draggablepanel/src/main/java/com/github/pedrovgs/DraggablePanel.java

```diff
@@ -144,6 +146,38 @@ public void setClickToMinimizeEnabled(boolean enableClickToMinimize) {
   }
 
   /**
+   * Return if touch listener is enable or disable
+   */
+  public boolean isEnableTouchListener() {
+    return this.enableTouchListener;
+  }
```

Comment:
You don't need this method, this variable should not be accessed out of this class.